client/state/automated-transfer/selectors.js

```diff
@@ -68,3 +69,21 @@ export const isEligibleForAutomatedTransfer = compose(
 	getEligibilityStatus,
 	getEligibility
 );
+
+/**
+ * Determine if eligibility warnings or holds exist for
+ * a site other than a business plan hold.
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for transfer info
+ * @returns {boolean} False if there is one and only one hold/warning referring to missing business plan
+ */
+export const hasEligibilityMessagesOtherThanBusiness = ( state, siteId ) => {
```

Comment:
> Our actual goal, however, is to know when to show the eligibility card in the theme upload page.

there is a comment I left in the related PR where you have introduced the selector separately. it sounds an awful lot like this is `render()` logic and not state logic and would probably be best left separate from the state tree.